components/http/sources/raw-http-webhook-requests/raw-http-webhook-requests.js

```diff
@@ -0,0 +1,78 @@
+const httpApp = require("../../http.app");
+
+// Core HTTP component
+module.exports = {
+  key: "http-new-requests",
```

Comment:
`key` should follow the standard component naming convention - https://pipedream.com/docs/components/guidelines/#guidelines-patterns. While we're at it, let's remove `raw-` in the file and folder name. The props render that label unnecessary.